experiments/AMIP/modular/components/ocean/slab_ocean_init.jl

```diff
@@ -2,6 +2,7 @@ using ClimaCore
 
 import ClimaCoupler.Interfacer: OceanModelSimulation, get_field, update_field!
 import ClimaCoupler.FieldExchanger: step!, reinit!
+import ClimaCoupler.FluxCalculator: update_turbulent_fluxes_point!, surface_thermo_state
```

Comment:
`surface_thermo_state` isn't used in this file anymore

Reply:
True, not anymore ;-)